charts/osm/crds/meshconfig.yaml

```diff
@@ -83,12 +83,16 @@ spec:
                       default: false
                     outboundIPRangeExclusionList:
                       description: Global list of IP address ranges to exclude from outbound traffic interception by the sidecar proxy.
-                      type: string
-                      pattern: ((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2})(,((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2}))*$
+                      type: array
+                      items:
+                        type: string
+                        pattern: ((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2})$
```

Comment:
why did the pattern change ?

Reply:
https://github.com/openservicemesh/osm/blob/main/pkg/apis/config/v1alpha1/mesh_config.go#L37

The CRD definition actually should be changed initially. After changing configmap to MeshConfig `outboundIPRangeExclusionList` does not need to be a comma separated string. So changing it to an array of CIDRs